### PR TITLE
Livestream instructions

### LINKED ISSUE #728: Add basic documentation for video livestreams
Well before we go live, we should at least have basic documentation on how we are doing this on About. This can of course be refined over time, but it should be easy for all staff (and curious community members) to see what and how we are doing this.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #728 

Depends on #734  

-----
[View rendered activities/index.md](https://github.com/publiccodenet/about/blob/livestream/activities/index.md)
[View rendered activities/live-streaming/index.md](https://github.com/publiccodenet/about/blob/livestream/activities/live-streaming/index.md)
[View rendered activities/live-streaming/manage-live-stream.md](https://github.com/publiccodenet/about/blob/livestream/activities/live-streaming/manage-live-stream.md)
[View rendered activities/live-streaming/post-production.md](https://github.com/publiccodenet/about/blob/livestream/activities/live-streaming/post-production.md)
[View rendered activities/live-streaming/prepare-live-stream.md](https://github.com/publiccodenet/about/blob/livestream/activities/live-streaming/prepare-live-stream.md)

### CODE REVIEW

File: activities/live-streaming/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+# Live streamed podcast
+
+We use live streams as a tool to communicate. The format we have chosen is a podcast style live stream and we call the show **Let's talk about Public Code**.
+
+## Goals
+
+- **Become discoverable:** Create another way for people we would like to know who don't know that we exist or what we do *to discover us*.
+- **Grow** our codebases **community**.
```
Comment (clausmullie) [inline activities/live-streaming/index.md:12]: I think this should be either:
singular - codebase's community
plural - codebases' communities
and probably the plural?

File: activities/live-streaming/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Get in **direct touch with key figures** of top projects. Possible way in to steward or make them members of the Foundation for Public Code.
```
Comment (clausmullie) [inline activities/live-streaming/index.md:13]: I would rephrase to 'get in touch directly'

**Comment by clausmullie** [comment]
General comment - inconsistent sfull tops in bullet lists. My understanding is that we do not put full stops at the ends of bullets, but I may be wrong.

**Comment by Ainali** [comment]
> General comment - inconsistent sfull tops in bullet lists. My understanding is that we do not put full stops at the ends of bullets, but I may be wrong.

Nice catch! Fixed!

File: activities/live-streaming/post-production.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+type: guide
+explains: how to do post-production a live stream
+---
+
+# Post-producing a live stream
+
+When doing post-production of a 'Let's talk about  public code' [live stream](index.md), there are some steps to remember.
+
+- Follow up of attendees/viewers: connect them to a Call to Action such as [sign up for our tailored-to-your-preferences-newsletter](https://forms.gle/gn7wR2Eaxbv5g1BF9).
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:10]:
```suggestion
- Attendee and viewer follow up: connect them to a Call to Action such as [sign up for our tailored-to-your-preferences-newsletter](https://forms.gle/gn7wR2Eaxbv5g1BF9).
```

File: activities/live-streaming/post-production.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Cut an audio version and upload to [Podbean](../tool-management/podbean.md).
+  - Cut out log pauses, coughs and other noise, but try to keep the content similiar to the recording.
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:12]:
```suggestion
  - Cut out log pauses, coughs and other noise, but try to keep the content similar to the recording.
```

File: activities/live-streaming/post-production.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  - Compress and normalize.
+  - Cut out the long intro audio for a shorter audio intro.
+  - Change the outro audio for a complete version.
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:15]: What does 'a complete version' mean?

File: activities/live-streaming/post-production.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+- Post a blogpost with the link to the video and also the audio.
+- Social media campaign ending.
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:17]: Does this need a verb? Does it include publicly thanking the guest again?

File: activities/live-streaming/post-production.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+When doing post-production of a 'Let's talk about  public code' [live stream](index.md), there are some steps to remember.
+
+- Follow up of attendees/viewers: connect them to a Call to Action such as [sign up for our tailored-to-your-preferences-newsletter](https://forms.gle/gn7wR2Eaxbv5g1BF9).
+- Cut an audio version and upload to [Podbean](../tool-management/podbean.md).
+  - Cut out log pauses, coughs and other noise, but try to keep the content similiar to the recording.
+  - Compress and normalize.
+  - Cut out the long intro audio for a shorter audio intro.
+  - Change the outro audio for a complete version.
+- Post a blogpost with the link to the video and also the audio.
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:16]:
```suggestion
- [Post a blogpost](../communications/blogging.md) with the link to the video and the audio.
```

File: activities/live-streaming/post-production.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+- Social media campaign ending.
+- Some time after the show, take a look at the metrics (number of views, number of people that signed up, increase on number of leads that reach out to us later on, number of clicks in our blogposts, videos, audios, social media, etc.). Iterate, engage and act according to prior data.
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:18]:
```suggestion
- Some time after the show, take a look at the metrics (number of views, number of people that signed up, increase in number of leads that reach out to us later on, number of clicks in our blogposts, videos, audio, social media, etc.). Iterate, engage and act according to prior data.
```

File: activities/live-streaming/post-production.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+- Request feedback from the audience and the guest.
```
Comment (ElenaFdR) [inline activities/live-streaming/post-production.md:19]: Suggestion: make this: 'send guest a thank you email with request for feedback'. Move requesting audience feedback to, say, the blogpost or end of the podcast.

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
Sorry to leave this for so long! It looks like there are a lot of changes, but most are only stylistic or proofreading - please don't be daunted. Happy to chat if there are any questions, of course.

File: activities/live-streaming/post-production.md (hunk 5 above)
Comment (Ainali) [reply activities/live-streaming/post-production.md:15]: In the live stream, it's almost impossible to time the audio to stop at the same time as the stream, so it is usually cut off too early. This step intends to have the audio run to its natural end.

File: activities/live-streaming/manage-live-stream.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+- Prepare tabs with easy access to moderation tools of the different platforms to be able to intervene in case of offensive commenting. Preferably, this is done by a staff member not in the actual interview.
+- Meet with the guest in [StreamYard](../tool-management/streamyard.md] 30 minutes before going live
+- Keep an eye on attendees' questions and behaviour: when are they connecting, which is the most successful platform and why, best timing, etc.
+- When it's time to start the stream, press Go live and quickly start the intro video
+- Start by only having the interviewers on screen.
+- Remind the audience what the Foundation for Public Code does and how to contact us in an easy way.
+- Present the guest and bring them on screen.
+- Run the interview.
+- Bring up interesting questions from the audience on the screen and read them out loud to the interviewee.
+- Remember that there will be a podcast version afterwards, so make sure to explain things so that someone only listening to the show will understand (in case of shared screens or funny things happening on camera).
```
[the remark this answers is not in the document]
Comment (Ainali) [reply activities/live-streaming/manage-live-stream.md:20]: I think that is important, and there is a reminder. Or is it too poorly phrased?

File: activities/live-streaming/post-production.md (hunk 6 above)
Comment (Ainali) [reply activities/live-streaming/post-production.md:17]: Hmm. Yes, it needs something more. @AlbaRoza I copied this from your notes, how can we clarify this?

**Review by bvhme — DISMISSED** [review]

File: activities/live-streaming/prepare-live-stream.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+- Create a new preview image adding the guest and organisation with the help of this [template](preview-image.svg)
+- Earliest a week before the stream, create a new broadcast in [StreamYard](../tool-management/streamyard.md)
+- Upload the new intro video in Streamyard, remove the old one to avoid confusion during the stream
+- Update banners with links related to the upcoming show
+
+## Communication
+
+Ready **three weeks** (at least) in advance:
+
+- Copies
```
[the remark this answers is not in the document]
Comment (AlbaRoza) [reply activities/live-streaming/prepare-live-stream.md:31]: I meant the stuff that is listed below: copies (texts we are going to need), images and or graphs, etc.

File: activities/live-streaming/prepare-live-stream.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+- Images
+- Graphs
```
[the remark this answers is not in the document]
Comment (AlbaRoza) [reply activities/live-streaming/prepare-live-stream.md:33]: In Communications and also the resources needed in the development of the livestream per se.

**Review by AlbaRoza — COMMENTED** [review]
Created a [pull request](https://github.com/publiccodenet/about/pull/778) with suggested changes.

File: activities/live-streaming/prepare-live-stream.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (Ainali) [reply activities/live-streaming/prepare-live-stream.md:31]: Ah, you meant 'copy' as in text? Perhaps we should change this to 'script' instead to be more clear?

[the remark this answers is not in the document]
Comment (AlbaRoza) [reply activities/live-streaming/prepare-live-stream.md:31]: Yep, copy as in [copywriting](https://en.wikipedia.org/wiki/Copy_(written))

**Review by bvhme — DISMISSED** [review]

File: activities/live-streaming/index.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+- Get in **touch directly with key figures** of top projects. Possible way in to steward or make them members of the Foundation for Public Code.
+- Get **more** Social Media **followers** since we are going to use those networks for this project.
+
+## Formats, length and schedule
+
+1. Frequency: Ambition to make it at least monthly.
+2. Format: Two different formats: one for codebases, one for the Foundation. For the codebases it is an interview made by two codebase stewards - ideally the community ones - with someone relevant from a codebase. For the Foundation, the format is a conversation between two persons from the foundation. This can be theoretical, about ideas and principles or about news.
+3. Language: English.
+4. Script: We get several questions ready before each episode. This builds familiarity with our approach and can also reveal similarities and differences between codebases.
+5. Lenght and schedule: Minimum 20 minute livestream during business hours - starting at 14.00 UTC. Promise that the podcast will be live by close of business the next day. Aim to hold this on Tuesdays.
```
Comment (ElenaFdR) [inline activities/live-streaming/index.md:22]:
```suggestion
5. Length and schedule: Minimum 20 minute livestream during business hours - starting at 14.00 UTC. Promise that the podcast will be live by close of business the next day. Aim to hold this on Tuesdays.
```

**Review by AlbaRoza — APPROVED** [review]
Looks ready to go!

**Comment by ElenaFdR** [comment]
@Ainali @AlbaRoza Hey, it looks like you ignored or unfixed a bunch of my November edits, and then merged your own work today. What happened?

**Comment by Ainali** [comment]
Oh, if we did it was a mistake! Which commits was it?

**Comment by ElenaFdR** [comment]
Looking at it commit-by-commit, I was inconsistent in catching all instances of the same problem. I'll fix them on the release branch.
